sdks/python/apache_beam/ml/inference/pytorch_inference_test.py

```diff
@@ -115,12 +115,12 @@ def forward(self, x):
     return out
 
 
-class PytorchLinearRegressionKwargsPredictionParams(torch.nn.Module):
+class PytorchLinearRegressionKeyedBatchAndExtraParams(torch.nn.Module):
   """
-  A linear model with kwargs inputs and non-batchable input params.
+  A linear model with batched keyed inputs and non-batchable extra params.
 
-  Note: k1 and k2 are batchable inputs passed in as a kwargs.
-  prediction_param_array, prediction_param_bool are non-batchable inputs
+  Note: k1 and k2 are batchable examples passed in as a keyed to torch dict.
```

Comment:
"a keyed to torch dict"?

Reply:
Yeah, that'll work

Reply:
Sorry, I misread this comment. I'll reword.

Reply:
Fixed.